includes/admin/class-wc-rest-payments-onboarding-controller.php

```diff
@@ -183,7 +183,7 @@ public function get_progressive_onboarding_eligible( WP_REST_Request $request )
 			[
 				'business_info'   => $request->get_param( 'business' ),
 				'store_info'      => $request->get_param( 'store' ),
-				'woo_store_stats' => $request->get_param( 'woo_store_stats' ),
+				'woo_store_stats' => (array) $request->get_param( 'woo_store_stats' ),
```

Comment:
Nitpick: What do you think about using the null coalesce `??` operator to make it more clear that we would expect this param to already be an array, it's only if it's null we want to send an empty array. For example:

```php 
'woo_store_stats' => $request->get_param( 'woo_store_stats' ) ?? [],
```

Alternatively, we can just not send `woo_store_stats` at all in the request if it's null, because it is an optional parameter anyway.

Reply:
Great suggestion! Fixed in https://github.com/Automattic/woocommerce-payments/pull/6374/commits/cf83d7b4f0c7a90e5c0e5b99138c57cb5857c539